packages/remote-config/android/src/main/java/io/invertase/firebase/config/UniversalFirebaseConfigModule.java

```diff
@@ -143,9 +147,53 @@ Task<FirebaseRemoteConfigInfo> ensureInitialized(String appName) {
     return ensureInitializedTask;
   }
 
+  void onConfigUpdated(String appName, Callback callback) {
+    FirebaseApp firebaseApp = FirebaseApp.getInstance(appName);
+
+    FirebaseRemoteConfig.getInstance(firebaseApp).addOnConfigUpdateListener(new ConfigUpdateListener() {
```

Comment:
This event listener should probably be tracked like we do with auth [here](https://github.com/invertase/react-native-firebase/blob/main/packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java#L96) and removed using [this overridden method](https://github.com/invertase/react-native-firebase/blob/main/packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java#L117). This means you will have to move it to the `ReactNativeFirebaseConfigModule`.  Remote Config has multi-app support so you will need to track them on a per Firebase app basis so the implementation shouldn't be too far removed from how we implement the authStateListener [here](https://github.com/invertase/react-native-firebase/blob/main/packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java#L152-L181). i.e. use the app name as a key for the listener.